### ISSUE TITLE
Commits on Feb 1, 2020 LEDs not changing during start and stop of a race

### ISSUE THREAD

**Issue by BoLtWD** [RotorHazard/RotorHazard#231]
Commits on Feb 1, 2020

I don't know if its just me but after above comment update the Leds signal  Server Startup then stay on green and do not change though a race has started. After pressing stop the leds goes off but again if you start a race they do nothing.
I have manual control of Leds using the led control panel on race page

**Below is start up of the sever**
![lednotstarting](https://user-images.githubusercontent.com/25492262/73594360-bd38f100-4505-11ea-9646-e324bcc20ee8.jpg)

**Below output during GUI connection to server and running a race, recording pass and stopping a race**
![lednot](https://user-images.githubusercontent.com/25492262/73594543-3422b980-4507-11ea-99db-d7d3cef31eb0.jpg)

**Comment by BoLtWD** [comment]
I have also updated to Commits on Feb 1, 2020 but the issue is still the same

**Comment by BoLtWD** [comment]
Below is my config.json

{
	"GENERAL": {
		"HTTP_PORT": 5000,
		"ADMIN_USERNAME": "admin",
		"ADMIN_PASSWORD": "RotorHazardiscool",
		"DEBUG": false,
		"CORS_ALLOWED_HOSTS": "*"
	},
	"SENSORS": {
	},
	"SERIAL_PORTS": [],
	"LED": {
		"LED_COUNT": 150,
		"LED_PIN": 10,
		"LED_FREQ_HZ": 800000,
		"LED_DMA": 10,
		"LED_INVERT": false,
		"LED_CHANNEL": 0
	}
}

**Comment by HazardCreative** [comment]
Have you been to the LED setup section in Settings? Can you assign effects to different events? Do those effects work when you select them directly in LED Control on the Race page?

**Comment by BoLtWD** [comment]
Discard issue found, I had done something during updating and I got it working by using a back up and re-updating to Commits on Feb 1, 2020.

_@HazardCreative On 1st start up Leds signal Server Startup and end stuck on green (don't go off), due to this when you start a race the leds do not signal Race Staging, then Race Start. When you stop the race the green leds are turned off and when you restart a race the leds start signaling Race Staging, then Race Start, passes and stop._

**In short the issue has changed to text in italic**

**Comment by BoLtWD** [comment]
Ok that was strange as now all is ok like it just fixed itself. Shot in the dark Is the led event settings held in the browser cash and if so could it be due to this?

I have repowerd the timer many times and back to running a race, changing event leds and rerunning a race and found this issue has also gone. Think I need a beer now..

**Comment by ethomas997** [comment]
@BoLtWD There was a bug that was causing the weird LED handlers issues; fixed by https://github.com/RotorHazard/RotorHazard/commit/df3798ff3f0827fa34456211d6b85be7769eb159
`After any LED handler entries were customized in Setup, only customized entries would be enabled after next restart.`
